OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -44,6 +44,7 @@ public class LobbyCommands : ServerTrait, IInterpretCommand, INotifyServerStart,
 			{ "faction", Faction },
 			{ "team", Team },
 			{ "spawn", Spawn },
+			{ "toggle-spawn", ToggleSpawn },
```

Comment:
```suggestion
			{ "toggle_spawn", ToggleSpawn },
```

An underscore, instead of a dash, is consistent with the existing keys.